Ginger/GingerCore/Drivers/Common/HTMLElementInfo.cs

```diff
@@ -16,7 +16,10 @@
 */
 #endregion
 
+using Amdocs.Ginger.Common;
+using Amdocs.Ginger.Common.Repository;
 using Amdocs.Ginger.Common.UIElement;
+using Amdocs.Ginger.Repository;
 using HtmlAgilityPack;
```

Comment:
let's remove those using if not needed anymore

Reply:
Removed